src/main/webapp/WEB-INF/jsp/application/include/app-detail-elements/app_progress.jsp

```diff
@@ -28,7 +28,14 @@
                     />
                 </td>
                 <td>
-                    <c:out value="${comment.person.niceName}"/>
+                    <c:choose>
```

Comment:
This part must be replaced after jsp deletion (we can use the part like `sick_note.html`)